pyvista/plotting/camera.py

```diff
@@ -297,8 +298,58 @@ def direction(self):
         --------
         >>> import pyvista
         >>> plotter = pyvista.Plotter()
-        >>> plotter.camera.direction  # doctest: +SKIP
+        >>> plotter.camera.direction = (0.0, 0.0, -1.0)
+        >>> plotter.camera.direction
         (0.0, 0.0, -1.0)
 
         """
         return self.GetDirectionOfProjection()
+
+    @direction.setter
+    def direction(self, direction):
+        """Set the vector from the camera position to the focal point.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> plotter = pyvista.Plotter()
+        >>> plotter.camera.direction = (0.0, 0.0, -1.0)
+
+        """
```

Comment:
Needs code?

Reply:
Yea, code on setters isn't necessary and won't be shown in sphinx.

Reply:
No this setter is not need.